contrib/automation_tests/test_cases/symbols_tab.py

```diff
@@ -18,21 +23,229 @@
 from test_cases.capture_window import Capture
 from test_cases.live_tab import VerifyFunctionCallCount
 
+Module = namedtuple("Module", ["name", "path", "is_loaded"])
+CACHE_LOCATION = "{appdata}\\OrbitProfiler\\cache\\".format(appdata=os.getenv('APPDATA'))
+
 
 def _show_symbols_and_functions_tabs(top_window):
     logging.info("Showing symbols tab")
     find_control(top_window, "TabItem", "Symbols").click_input()
 
 
-class LoadSymbols(E2ETestCase):
+def _find_and_close_error_dialog(top_window) -> str or None:
+    window = find_control(top_window, 'Window', 'Error loading symbols', raise_on_failure=False)
+    if window is not None:
```

Comment:
I would flip this if to early return. `if window is None -> return`, then the big part of the if does not need to be indented

Reply:
Changed, thanks